models/Content.js

```diff
@@ -1,9 +1,15 @@
 const mongoose = require('mongoose');
 
 const contentSchema = new mongoose.Schema({
-  files: {
-    type: [String],
-    required: true,
+  file: {
+    name: {
+      type: String,
+      required: true,
+    },
+    buffer: {
+      type: mongoose.Schema.Types.Buffer,
+      required: true,
+    },
   },
   title: {
```

Comment:
What is this title?
Isn't the file's name enough?

In the previous model we had several files (an array), here we just have content=file, is this what we want?

Reply:
The title is a title for the content (for example, "Smartphones Lecture").
The file name is the actual file name (for example, "smartphones_lecture.pdf").

I think it's OK to have one file for each "content".

Reply:
I think we should give them the ability to upload multiple files.
If you want to split the lecture to 5 episodes that each one represent by a PDF file?

Reply:
OK, I'll do that.